ios-base/build_kotlin_modules.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd ../
+./gradlew :ios-combined:linkIosX64 -PXCODE_CONFIGURATION=${CONFIGURATION}
```

Comment:
XCODE_CONFIGURATION looks better than CONFIGURATION

Reply:
I'm just curious about how many times we should execute this script. Is running once enough basically? 👀

Reply:
> XCODE_CONFIGURATION looks better than CONFIGURATION

I'm sorry, it doesn't work. 🙏 I will set configuration by variable `XCODE_CONFIGURATION `


> I'm just curious about how many times we should execute this script. Is running once enough basically? 👀

I think this script should execute once enough basically. When Kotlin module is changed, we should execute this script.

Reply:
This is no longer needed by https://github.com/DroidKaigi/conference-app-2020/pull/447/commits/2111bd7f13b0ac2963a14c516488dd49aeacc75d 🙏

Reply:
I guess modifying Kotlin modules will happen especially in the early stage. To avoid some troubles, I suppose it would be nicer if this script would be executed in the build phases like cocoapods' linking. What do you think about it? I'm just an iOS newbie so I can only say from my imagination though...

Reply:
ah, make sense. cool.